browser-app/package.json

```diff
@@ -42,7 +41,8 @@
     "download:plugins": "theia download:plugins -p=true"
   },
   "engines": {
-    "node": ">=10.2.0 < 13"
+    "yarn": "1.0.x || >=1.2.1",
+    "node": ">=12.14.1 <13"
```

Comment:
Does it mean we don't support version 10 anymore at all?

Reply:
> Does it mean we don't support version 10 anymore at all?

Correct, the [framework dropped](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md#1_5_0_drop_node_10_support) support for `node 10` in the `1.5.0` release as 12 is the LTS.